velox/vector/tests/VectorMaker.h

```diff
@@ -623,6 +623,12 @@ class VectorMaker {
 
   static VectorPtr flatten(const VectorPtr& vector);
 
+  ArrayVectorPtr createArrayVector(
```

Comment:
Other similar methods in this class are called arrayVector. Let's rename this method to match and move it next to other similar methods. It would also be nice to document this method.